### PR TITLE
Update to JDA 5 & bump to JDA-Chewtils 2.0

### PR DESCRIPTION
**Pull request by rtm516**
[contributing]: https://github.com/DV8FromTheWorld/JDA/wiki/5%29-Contributing
[pull-request]: https://github.com/JDA-Applications/JDA-Utilities/pulls

#### Pull Request Checklist
Please follow the following steps before opening this PR.<br>
PRs that do not complete the checklist will be subject to denial for
missing information.

- [x] I have checked the [pull request page][pull-request] for upcoming
      or merged features/bug fixes.
- [x] I have read JDA's [contributing guidelines][contributing].

#### Pull Request Information
Check and fill in the blanks for all that apply:

- [x] My PR fixes a bug, error, or other issue with the library's codebase.
- [ ] My PR is for the `______` module of the JDA-Utilities library.
- [ ] My PR creates a new module for the JDA-Utilities library: `______`.

#### Description

Update to JDA v5

### CODE REVIEW

File: examples/src/main/java/com/jagrosh/jdautilities/examples/command/ServerinfoCommand.java
Comment (rtm516) [inline examples/src/main/java/com/jagrosh/jdautilities/examples/command/ServerinfoCommand.java]: > Guilds no longer have the Region option. Use VoiceChannel.getRegion() instead.

Because of this change we have to have this work around, it will throw an error if there is no voice channels.
Maybe worth just removing the line entirely?

File: build.gradle
Comment (rtm516) [inline build.gradle]: This defaults to build 0 so building locally doesn't break

Comment (Chew) [reply build.gradle]: I always put this as "DEV" so to not interfere with production builds.

File: examples/src/main/java/com/jagrosh/jdautilities/examples/command/ServerinfoCommand.java
Comment (Chew) [reply examples/src/main/java/com/jagrosh/jdautilities/examples/command/ServerinfoCommand.java]: Yeah since this is _server info_ not really needed plus this would almost always return automatic which would not have an emoji.

File: build.gradle
Comment (Andre601) [reply build.gradle]: Since this build would rely on an alpha of JDA, wouldn't it also make sense to like manually add a `alpha` or similar to the version here?

Comment (Chew) [reply build.gradle]: I'll likely just release `2.0-SNAPSHOT`s until a full release.

Comment (rtm516) [reply build.gradle]: Added `-SNAPSHOT`

**Comment by Chew** [comment]
For those wanting to test, this PR has been released as `2.0-SNAPSHOT`

I wanna put out a 1.24.0 before this is merged, so this is on hold for a bit (unless you or someone wants to fix the deprecated stuff in CommandEvent to use JDA _4_ methods in a separate PR for v1)


```xml
        <repository>
            <id>chew-m2-snapshots</id>
            <url>https://m2.chew.pro/snapshots/</url>
        </repository>
```
```xml
        <dependency>
            <groupId>pw.chew</groupId>
            <artifactId>jda-chewtils</artifactId>
            <version>2.0-SNAPSHOT</version>
            <scope>compile</scope>
            <type>pom</type>
        </dependency>
```

**Review by Chew — APPROVED** [review]
Good work so far!

**Pull request merged**
